include/maths/CInformationCriteria.h

```diff
@@ -42,9 +42,11 @@ LOG_DETERMINANT(5);
 #undef LOG_DETERMINANT
 
 //! The log determinant of our internal heap symmetric matrix.
+MATHS_EXPORT
```

Comment:
Why do we need the export in individual functions?

Reply:
Because these are free functions. See [here](https://docs.microsoft.com/en-gb/cpp/cpp/dllexport-dllimport?view=vs-2017) for more info.